### DISCUSSION TITLE
win环境

### DISCUSSION THREAD

**Discussion by Metasymbols** [AlistGo/alist#3309] [Q&A]
编译过程中，从github拉取包总是失败，网络不稳定，有没有解决办法

**Comment by anwen-anyi** [comment]
https://www.yuque.com/anwenya/alist/glqlhu#vDd2H 可能对你有帮助~

**Comment by Metasymbols** [comment]
谢谢，问题已经解决，刚安装的go环境，要通过配置代理网站拉取git依赖包才行

**Reply by anwen-anyi** [reply]
两个办法
 1. 配置代理
 2. 改回国内的源 
我上面提到的是方法2~
